feature/settings/src/main/res/values/strings.xml

```diff
@@ -83,5 +83,24 @@
     <string name="stabilization_description_on">Stabilization On</string>
     <string name="stabilization_description_high_quality">Stabilization High Quality</string>
     <string name="stabilization_description_off">Stabilization Off</string>
-    <string name="stabilization_description_unsupported">Stabilization unsupported</string>
+    <string name="stabilization_description_unsupported_device">Stabilization unsupported by device</string>
+    <string name="stabilization_description_unsupported_fps">Stabilization unsupported due to frame rate</string>
+
+
+    <!-- Target Fps setting strings -->
+    <string name="fps_title">Set Target Frame Rate</string>
```

Comment:
We should drop the "Target" word usage from the UI strings. In JCA, we want to always hit the target FPS, and should disable features if we can't. So if an FPS is set and the camera doesn't provide it, that should be considered a bug. We can keep the "Target" word in variable names, etc., but just don't use it in the UI.